### ISSUE TITLE
Are cluster-wide perms required on my referenced ServiceAccount for locked resources

### ISSUE THREAD

**Issue by komish** [redhat-cop/resource-locker-operator#13]
I'm doing some testing managing config maps similar to the documentation file that I submitted here: https://github.com/redhat-cop/resource-locker-operator/pull/11, but I'm seeing the operator logs complain about my ServiceAccount not having the ConfigMap permissions at a cluster scope[1]. Seems unexpected as I see the ConfigMap gets expected, but I want to be sure I'm understanding the model or otherwise want to report it as a bug.

[1]
```
E1219 21:25:55.551434       1 reflector.go:126] pkg/mod/k8s.io/client-go@v11.0.1-0.20190409021438-1a26190bd76a+incompatible/tools/cache/reflector.go:94: Failed to list *unstructured.Unstructured: configmaps is forbidden: User "system:serviceaccount:resource-locker-test-ns:resource-locker-test-sa" cannot list resource "configmaps" in API group "" at the cluster scope
```

**Comment by raffaelespazzoli** [comment]
the answer is yes because of this issue: https://github.com/kubernetes-sigs/controller-runtime/issues/671 it was documented at the end of the readme. It would be nice to get some help to work around that issue.

**Comment by raffaelespazzoli** [comment]
this is now fixed by #20
